modules/amazon/index.js

```diff
@@ -1,55 +1,50 @@
-const XRay = require('x-ray');
-const xray = XRay();
+const { URL } = require('url');
```

Comment:
I thought we turned off eslint's fetish for assignment by destructuring.

Reply:
In the case of requires I actually like it because it's reminiscent of the `import` syntax which should eventually replace `require`.

Reply:
🤷‍